lsp/nls/src/position.rs

```diff
@@ -122,8 +122,19 @@ impl PositionLookup {
             }
 
             match term.as_ref() {
-                Term::Fun(id, _) | Term::Let(id, _, _, _) => idents.push(*id),
-                Term::FunPattern(pat, _) | Term::LetPattern(pat, _, _) => {
+                Term::Fun(id, _) => idents.push(*id),
+                Term::Let(bindings, _, _) => {
+                    for (id, _) in bindings {
+                        idents.push(*id);
+                    }
```

Comment:
Alternatively, you can use something like:

```rust
idents.extend(bindings.iter().map(|(id, _)| *id))
```

While it's not necessarily way more readable, I suspect `extend` can use iterator size hints to allocate the required space at once (not that it'll make a huge difference here :man_shrugging: )